backend/clubs/views.py

```diff
@@ -2115,9 +2115,13 @@ def get_queryset(self):
         is_club_specific = self.kwargs.get("club_code") is not None
         if is_club_specific:
             qs = qs.filter(club__code=self.kwargs["club_code"])
-            qs = qs.filter(Q(club__approved=True) | Q(club__ghost=True))
+            qs = qs.filter(
+                Q(club__approved=True) | Q(type=Event.FAIR) | Q(club__ghost=True)
+            )
         else:
-            qs = qs.filter(Q(club__approved=True) | Q(club__ghost=True) | Q(club__isnull=True))
+            qs = qs.filter(
+                Q(club__approved=True) | Q(type=Event.FAIR) | Q(club__isnull=True)
```

Comment:
You removed `Q(club__ghost=True)` here.

Removing this will hide events from clubs that have been approved, but changed their name or description. We want to make sure these events stay up.

Reply:
lol I'm glad I made a PR

Reply:
Yeah sorry I didn't even notice that I made that change. I'll add it back now.